generators/component/templates/functions.php

```diff
@@ -2,12 +2,27 @@
 
 namespace Flynt\Components\<%= nameUpperCamelCase %>;
 
-use Flynt\Features\Components\Component;
+use Flynt;
+use Flynt\Utils\Options;
 
 add_filter('Flynt/addComponentData?name=<%= nameUpperCamelCase %>', function ($data) {
-    add_action('wp_enqueue_scripts', function () {
-        Component::enqueueAssets('<%= nameUpperCamelCase %>');
-    });
-
     return $data;
 });
+
+// Flynt\registerFields('<%= nameUpperCamelCase %>', [
```

Comment:
This has already changed, should now be `Api::registerFields`. And make sure to add `use Flynt\Api;` to the statements above.